README.md

```diff
@@ -170,35 +113,105 @@ More information about the
 [audit service](https://wiki.duraspace.org/display/FF/Design+-+Audit+Service)
 is available on the Fedora wiki.
 
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| audit.enabled | Enables/disables audit triplestore service  | false |
+| audit.input.stream | Audit Service jms message stream | broker:topic:fedora |
+| audit.event.baseUri | The baseUri to use for event URIs in the triplestore. A `UUID` will be appended to this value, forming, for instance: `http://example.com/event/{UUID}` | http://example.com/event |
+| audit.triplestore.baseUrl| The base url for the external triplestore service | http://localhost:3030/fuseki/test/update |
+| audit.triplestore.authUsername| Username for basic authentication against triplestore | |
+| audit.triplestore.authPassword| Password for basic authentication against triplestore | |
+| audit.filter.containers |  A comma-delimited list of URIs to be filtered (ignored) by the audit service | http://localhost:8080/fcrepo/rest/audit | 
```

Comment:
Does this default value still make sense?

Reply:
Maybe it is confusing.  I'll make a note to replace with something else after merging.